lib/test/t_Triangular_std.cxx

```diff
@@ -104,6 +127,8 @@ int main(int, char *[])
       fullprint << "characteristic function=" << CF << std::endl;
       Complex LCF = distribution.computeLogCharacteristicFunction( point[0] );
       fullprint << "log characteristic function=" << LCF << std::endl;
+      for (UnsignedInteger j = 0; j < refValues[nTriangular].getSize(); ++j)
+        assert(distribution.computeCharacteristicFunction(u[j], refValues[nTriangular][j]));
```

Comment:
this doesnt compile
did you mean assert_almost_equal ?
the parenthesis of computeCharacteristicFunction is misplaced
same in Trapezoidal test

Reply:
I saw it. I pushed an unsaved file :-(
It should be fixed now...